.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,54 @@
+name: Integration tests
+
+on:
+  pull_request:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  get-head-commit-msg:
```

Comment:
Is it not possible to declare this job once in a `yaml` file and re-use it in different jobs?

Reply:
Jobs in a workflow are self contained, there are workarounds to reuse a job yet we can't avoid the duplication